traffic_ops/testing/api/v4/tc-fixtures.json

```diff
@@ -2817,8 +2817,8 @@
                             "serviceAddress": false
                         },
                         {
-                            "address": "127.0.0.13/30",
-                            "gateway": "127.0.0.1",
+                            "address": "128.0.0.13/30",
+                            "gateway": "128.0.0.1",
```

Comment:
Since there were issues in API v4 tests that were causing failures. Fix: https://github.com/apache/trafficcontrol/pull/6544/commits/e2b9c6d754f374a16a8d4779fcb63e56ebea9abb#diff-848f3f399dd9b0437c562368648f7acd64533910f7ee2740decb4ff00d92e380
Issue: https://github.com/apache/trafficcontrol/runs/5315051608?check_suite_focus=true

Reply:
That doesn't seem related - especially since those tests failed at all three API versions, but you only changed the v4 fixtures - which are run last, as well. The first problem I see in the APIv2 test logs is
```
    servers_test.go:39: Response:  edge2-cdn1-cg3   {[{server was created. success}]}
    servers_test.go:56: Failed to get at least one server by hostname 'edge2-cdn1-cg3'
```
and then it fails to delete that server, and fails to delete things associated with it, and then fails to create those things because they already exist. Each other API version failure looks similar, but it's hard to say. If a server with a hostname is created and then cannot later be filtered for in a GET, it seems like it caught a real bug in TO, and I can't understand why changing an IP address in a file not used by that test would fix that.

Reply:
I just verified that those IP changes are the only things necessary to make the tests pass or fail. So you're right, but here's the thing. The error it's hitting (in v4 anyway, idk what's going in v2 and v2 isn't broken at the tip of your branch so I'm just gonna ignore that for now) is:
```
--- FAIL: TestCacheGroups (5.36s)
    servers_test.go:567: could not create server 'atlanta-org-1': error requesting Traffic Ops: path 'https://localhost:6443/api/4.0/servers' gave HTTP error 400 Bad Request - error-level alerts: there exists a server with id 6315 on the same profile that has the same service address 127.0.0.17/30 - alerts: [{Text:there exists a server with id 6315 on the same profile that has the same service address 127.0.0.17/30 Level:error}]
```
That's with the `tc-fixtures.json` from master just updated to use `profileNames` instead of `profile` on servers. There are 2 servers in the fixtures that use `127.0.0.17` - `atlanta-org-1` and `atlanta-mid-17` and both use it as their service address, that's true. But, `atlanta-mid-17` has `"profileNames": ["MID2"]` and `atlanta-org-1` has `"profileNames": ["EDGE1"]`.

So changing these IP addresses makes the tests pass, but only because it's hiding a bug. Two servers aren't allowed to share the same IP service address ever, because TO always thinks that they share a Profile. I believe I have a comment somewhere else asking about how we wanna go about handling that situation, but in any case what is happening right now is too restrictive; two different servers need to be able to share a service IP address as long as other conditions aren't met.

Reply:
Changed (dropped trigger) based on Brennan's and mine offline discussion.